src/subscribers/redis-subscriber.ts

```diff
@@ -23,10 +23,18 @@ export class RedisSubscriber implements Subscriber {
      * @return {void}
      */
     subscribe(callback): Promise<any> {
+        var _this   = this;
+
         return new Promise((resolve, reject) => {
             this._redis.on('pmessage', (subscribed, channel, message) => {
                 message = JSON.parse(message);
 
+                if (_this.options.devMode)
+                {
+                    Log.info(channel);
```

Comment:
Maybe we should prefix these as `'Channel: ' + channel` and  `'Event: ' + event`